### DISCUSSION TITLE
Feature for password generator - "Group similar characters"

### DISCUSSION THREAD

**Discussion by RomekJ** [dani-garcia/vaultwarden#5166] [Ideas]
Hello,

I use vaultwarden at home and I am very happy with it.

However, there is one option that I miss in the "Password Generator" function.

Please check / consider / maybe eventually add the "Group similar characters" option on the password generation screen.

**Example**:
Currently: Password 10 characters: "a-z", "A-Z", "0-9": sample result: **5z26qeQ9M6**
After the change: Password 10 characters: "a-z", "A-Z", "0-9", "group similar characters": sample result: **NS85949kev**

Such passwords are useful for services where I log in e.g. on a TV or in a car, where entering a password without a physical keyboard is not convenient.

Such an option was e.g. in the aWallet program for Android.

What do you think about it?

PS.
Is this a good place to report such a development?
If not, where can I report it?

Best regards!

**Comment by cure1245** [comment]
I'm going to have to respectfully disagree. If a service is important enough to have a randomly generated password, reducing the password space—and therefore the entropy—is not a good idea at all.

If a service isn't important enough for a random password, then you can use whatever low to medium strength password you normally use that is easy to memorize.

**Comment by BlackDex** [comment]
It's also not something this project can fix. This is a client feature which can't be controlled from the server side. 

These kind of features should be reported at Bitwarden.
